packages/devtools_app/lib/src/shared/analytics/_analytics_web.dart

```diff
@@ -749,17 +737,6 @@ void waitForDimensionsComputed(String screenName) {
   });
 }
 
-// Loading screen from a hash code, can't collect GA (if enabled) until we have
-// all the dimension data.
-void setupAndGaScreen(String screenName) {
-  if (!_analyticsComputed) {
-    _stillWaiting++;
-    waitForDimensionsComputed(screenName);
-  } else {
-    screen(screenName);
-  }
-}
-
```

Comment:
generally curious since the `_analytics_web.dart` file is imported through conditional imports where it is used in DevTools - does the check_unused_code command work well with conditional imports?

Reply:
I've fixed almost all false-positives with conditional imports (the only one left is when one conditionally import uses publicly available code from its file and another (main) does not). Unfortunately, analyzer resolves conditional imports not the way I'd expected it to be, it's calculated manually on top.

But as for this change, this code is unused. Or am I missing something?

Reply:
This code appears to be unused 👍